doc/deployment-on-arbitrary-cluster.md

```diff
@@ -0,0 +1,30 @@
+# Deployment on Arbitrary Cluster
+
+In this guide we will cover installation of Multus and
```

Comment:
It does not cover multus installation (and should not)

Reply:
Done